httpserver/http_connection.cpp

```diff
@@ -291,6 +321,12 @@ void connection_t::process_request() {
 
             BOOST_LOG_TRIVIAL(trace) << "swarms/push";
 
+            if (!verify_signature()) {
+                response_.result(http::status::bad_request);
+                response_.body() = "Could not validate batch signature";
```

Comment:
In this case it is not a batch, just a single message.

Reply:
Fixed